src/store/modules/search.store.js

```diff
@@ -152,6 +153,39 @@ const actions = {
                     coordinates: entry.coordinate,
                     dispatcher: dipsatcherSelectResultEntry,
                 })
+                break
+            case SearchResultTypes.FEATURE:
+                if (entry.extent.length === 2) {
+                    dispatch('zoomToExtent', { extent: entry.extent, dispatcher })
+                } else if (entry.zoom) {
+                    dispatch('setCenter', {
+                        center: entry.coordinate,
+                        dispatcher: dipsatcherSelectResultEntry,
+                    })
+                    dispatch('setZoom', {
+                        zoom: entry.zoom,
+                        dispatcher: dipsatcherSelectResultEntry,
+                    })
+                }
+
```

Comment:
this is the exact same code as the first part of the location code, which makes sense. Perhaps we could add a small 'zoomToEntry' function here, to avoid code repetition.